src/averageAges.js

```diff
@@ -15,11 +16,15 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const men = century
+    ? people.filter((p) => p.sex === 'm' && Math.ceil(p.died / 100) === century)
+    : people.filter((p) => p.sex === 'm');
+
+  let sum = 0;
+
+  men.forEach((m) => sum += m.died - m.born);
+
+  return sum/men.length;
```

Comment:
You should avoid forEach, for example you can use reduce. For readability I would add spaces around the division. You can also add rounding, so that there are no large values after the dot
```suggestion
  const meanAge = men
    .reduce((sum, m) => sum + m.died - m.born, 0);

  return Math.round((meanAge / men.length) * 100) / 100;
```